src/commands/test.js

```diff
@@ -1,525 +1,63 @@
-          standaloneNodeArgs,
-          nodeOutputChunks,
-        )
-      } catch (e) {
-        toolbox.print.error('')
-        toolbox.print.error('  Graph Node')
-        toolbox.print.error('  ----------')
-        toolbox.print.error(
-          indent('  ', Buffer.concat(nodeOutputChunks).toString('utf-8')),
-        )
-        toolbox.print.error('')
-        await stopTestEnvironment(tempdir)
-        process.exitCode = 1
-        return
-      }
-    }
-
-    // Wait for Graph Node to come up
-    try {
-      await waitForGraphNode(timeout)
-    } catch (e) {
-      toolbox.print.error('')
-      toolbox.print.error('  Graph Node')
-      toolbox.print.error('  ----------')
-      toolbox.print.error(
-        indent(
-          '  ',
-          await collectGraphNodeLogs(tempdir, standaloneNode, nodeOutputChunks),
-        ),
-      )
-      toolbox.print.error('')
-      await stopTestEnvironment(tempdir)
-      process.exitCode = 1
-      return
-    }
-
-    // Run tests
-    let result = await runTests(testCommand)
-
-    // Bring down Graph Node, if a standalone node is used
-    if (nodeProcess) {
-      try {
-        await stopGraphNode(nodeProcess)
-      } catch (e) {
-        // do nothing (the spinner already logs the problem)
-      }
-    }
-
-    if (result.exitCode == 0) {
-      toolbox.print.success('✔ Tests passed')
-    } else {
-      toolbox.print.error('✖ Tests failed')
-    }
-
-    // Capture logs
-    nodeLogs =
-      nodeLogs || result.exitCode !== 0
-        ? await collectGraphNodeLogs(tempdir, standaloneNode, nodeOutputChunks)
-        : undefined
-    ethereumLogs = ethereumLogs ? await collectEthereumLogs(tempdir) : undefined
-
-    // Bring down the test environment
-    try {
-      await stopTestEnvironment(tempdir)
-    } catch (e) {
-      // do nothing (the spinner already logs the problem)
-    }
-
-    if (nodeLogs) {
-      toolbox.print.info('')
-      toolbox.print.info('  Graph node')
-      toolbox.print.info('  ----------')
-      toolbox.print.info('')
-      toolbox.print.info(indent('  ', nodeLogs))
+    const platform = getPlatform();
+    if (!version) {
+      let xmlHttp = new XMLHttpRequest();
+      xmlHttp.open('GET', 'https://api.github.com/repos/LimeChain/matchstick/releases/latest', false);
+      xmlHttp.send();
+      const jsonResponse = JSON.parse(xmlHttp.responseText);
+      version = jsonResponse.tag_name;
     }
 
-    if (ethereumLogs) {
-      toolbox.print.info('')
-      toolbox.print.info('  Ethereum')
-      toolbox.print.info('  --------')
-      toolbox.print.info('')
-      toolbox.print.info(indent('  ', ethereumLogs))
-    }
-
-    // Always print the test output
-    toolbox.print.info('')
-    toolbox.print.info('  Output')
-    toolbox.print.info('  ------')
-    toolbox.print.info('')
-    toolbox.print.info(indent('  ', result.output))
+    const url = `https://github.com/LimeChain/matchstick/releases/download/${ version }/${ platform }`;
```

Comment:
There's no need for the spaces here though, not sure if you added for readability.
You don't have to change if you don't want, we don't have a linter yet.